src/kirby/components/floating-action-button/floating-action-button.component.tns.scss

```diff
@@ -0,0 +1,30 @@
+// @import '../../scss/web-imports';
+@import '../../scss/native-imports';
+
+.fab {
+    background-color: get-color('kirby-primary');
+    border-radius: 32;
```

Comment:
Question.... shouldn't the size be externally defined, in a variable?

Reply:
I agree with you, they could easily be defined externally. However, I did this in correlation to the style guide in Zeplin and no variables were used there. Furthermore, we need to define them in the code and we don't have it yet.

Reply:
I think this is a great candidate for the guild meeting this morning